pkg/daemon/ceph/agent/flexvolume/controller.go

```diff
@@ -197,6 +198,17 @@ func (c *Controller) Attach(attachOpts AttachOptions, devicePath *string) error
 	return nil
 }
 
+// Resize RBD image
+func (c *Controller) Resize(expandArgs ExpandArgs, _ *struct{}) error {
```

Comment:
I'm struggling with terminology here. Resize suggests grow and shrink, but expand suggests only grow. If this only allows growing, what happens if this gets negative valaues?